wiki/mkdocs.yml

```diff
@@ -129,4 +129,5 @@ plugins:
 
     - redirects:
         redirect_maps:
-            '': 'https://gridsingularity.github.io/gsy-e/documentation/'
+            # When trying to reach the generic index page, redirect to the documentation page
+            'index.md': 'documentation.md'
```

Comment:
This will also work when deployed, right?

Reply:
There's only one way to know 😬 It should. In any case, it would be easy to revert it.